space_packet_parser/definitions.py

```diff
@@ -0,0 +1,14 @@
+# NOTE: This entire file is deprecated and here just to maintain backwards compatibility.
+# The functionality has been moved to space_packet_parser.xtce.definitions.XtcePacketDefinition
+import warnings
+
+from space_packet_parser.xtce import definitions
+
+
+class XtcePacketDefinition(definitions.XtcePacketDefinition):
+    def __init__(self, xtce_document, **kwargs):
```

Comment:
previously there was an ns keyword, that I wasn't entirely clear how to transition over to the new style. I don't use the ns mapping in any of my projects, so this at least helps me for now. But happy to add that capability in if someone can tell me what I need to do.

https://github.com/lasp/space_packet_parser/blob/66aef0906ab209aabf0bb97636ddc66e449b6a70/space_packet_parser/definitions.py#L66C13-L66C38

Reply:
The ns keyword was a way to pass in a namespace mapping dict but now that gets parsed automatically out of the XML document if the root element contains any `xmlns=<uri>` or `xmlns:prefix=<uri>` attributes. We're leaning more heavily on users to provide valid XML documents and manage their own namespace mappings, which goes hand in hand with #129